src/mappers/access-ticket-mapper.test.ts

```diff
@@ -9,6 +9,8 @@ describe('Mapping functions', () => {
       const entity: AccessTicketEntity = {
         id: '123',
         shlink_id: 'abc',
+        created_at: new Date('2024-01-01T00:00:00Z'),
+        updated_at: new Date('2024-01-01T00:00:00Z')
       };
```

Comment:
Hello @godchiSymbionix, I suggest to wrap up this into a variable, since you're using the exact same object later on in the assertion.